templates/vue/src/components/ModalButton.vue

```diff
@@ -81,15 +81,17 @@ export default {
   border-radius: 50%;
   width: 40px;
   height: 40px;
-  font-size: 18px;
```

Comment:
I think this is supposed to stay as `px` because it is a button with an icon. Icon-only buttons should not generally use `em` since they are not really text.

Reply:
I was keeping this consistent with master that's why it was changed. I personally think it's ok to keep it as `em`. `em` is not (and I don't think should be) limited to text only. I used it extensively in the sidebar PR so everything stays consistent and responsive.